appium/config/wdio.ios.app.conf.js

```diff
@@ -0,0 +1,33 @@
+const { join } = require('path');
+const { config } = require('./wdio.shared.conf');
+const pathWdioConfig = require('path');
+require('dotenv').config({ path: pathWdioConfig.resolve(__dirname, '../.env') });
+
+config.suites = {
+    all: [
+        './tests/specs/**/*.spec.ts'
+    ],
+    smoke: [
+        './tests/specs/login/GmailLogin.spec.ts',
+        './tests/specs/inbox/ReadTextEmail.spec.ts'
+    ]
+};
+
+config.capabilities = [
+    {
+        platformName: 'iOS',
+        iosInstallPause: 5000,
+        deviceName: 'iPhone 11 Pro Max',
+        platformVersion: '14.5',
```

Comment:
Would this run iOS 15 as well? We've just bumped app minimum ios requirement to 15.

Reply:
yeah, all we need is to replace platform version or even use ENV variable for that and run what you need, I will add ENV variable for this